content/id/docs/tasks/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Tugas (Tasks)
+main_menu: true
+weight: 50
+content_template: templates/concept
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+
+Bagian dokumentasi Kubernetes ini berisi halaman-halaman yang perlihatkan 
+bagaimana melakukan setiap tugas (_task_). Halaman tugas menunjukkan cara melakukan
+satu hal saja, biasanya dengan memberikan urutan langkah pendek.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Antarmuka Pengguna Berbasis Web (Dasbor)
+
+Melakukan _deploy_ dan mengakses dasbor (_dashboard_) berbasis web untuk 
+membantu kamu mengelola dan memantau aplikasi yang dimasukkan ke dalam kontainer
+di Kubernetes.
+
+## Menggunakan Baris Perintah kubectl
+
+Instalasi dan konfigurasi utilitas baris perintah `kubectl` yang digunakan untuk
+mengelola secara langsung klaster Kubernetes.
+
+## Mengkonfigurasi Pod and Kontainer
```

Comment:
```suggestion
## Mengkonfigurasi Pod and Container
```
Container here refer to https://kubernetes.io/docs/concepts/containers/
and we tried to change existing Kontainer to Container for k8s object in https://github.com/kubernetes/website/pull/18177
but it kinda tricky, discussed [here on slack channel](https://kubernetes.slack.com/archives/CJ1LUCUHM/p1576345141002200)